rmf_fleet_msgs/msg/RobotMode.msg

```diff
@@ -13,5 +13,11 @@ uint32 MODE_DOCKING=7
 uint32 MODE_ADAPTER_ERROR=8
 
 uint32 MODE_CLEANING=9
+uint32 MODE_PERFORMING_ACTION=10
+uint32 MODE_ACTION_COMPLETED=11
```

Comment:
Can we add comments describing why this is needed by `rmf_demos_fleet_adapter` and discourage the use of this in actual FA?

Reply:
[492ec60](https://github.com/open-rmf/rmf_internal_msgs/pull/70/commits/492ec609fb61a28cbc15dce4d80d15b523013dc5)